libhttpsimple-bindings/src/main/scala/libhttpsimple/HttpHeaders.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2017 Lightbend, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package libhttpsimple
+
+case class HttpHeaders(headers: Map[String, String]) {
```

Comment:
I think we should have documentation for the case-insensitive behaviour as ScalaDoc with example, i.e. `HttpHeaders("Cache-Control")` returns the same result as `HttpHeaders("cache-control")`.